addons/mail_client_extension/controllers/authenticate.py

```diff
@@ -38,9 +38,9 @@ def auth_confirm(self, scope, friendlyname, redirect, info=None, do=None, **kw):
             name = friendlyname if not info else f'{friendlyname}: {info}'
             auth_code = self._generate_auth_code(scope, name)
             # params is a MultiDict which does not support .update() with kwargs
-            params.update({'success': 1, 'auth_code': auth_code})
+            params.update({'success': 1, 'auth_code': auth_code, 'state': kw.get('state', '')})
```

Comment:
What is this change about ?

Reply:
This is needed for the gmail plugin. 
cc @std-odoo

Reply:
Could be worth a small comment then :)

Reply:
Comment added :)